gix-blame/src/file/function.rs

```diff
@@ -252,6 +199,24 @@ where
     })
 }
 
+/// The blobs storing the blamed file in `entry` and `parent_entry` are identical which is why
+/// we can pass blame to the parent without further checks.
+fn pass_blame_from_to(from: ObjectId, to: ObjectId, hunks_to_blame: &mut Vec<UnblamedHunk>) {
+    for unblamed_hunk in hunks_to_blame {
+        unblamed_hunk.pass_blame(from, to);
+    }
+}
+
+fn unblamed_to_out(hunks_to_blame: &mut Vec<UnblamedHunk>, out: &mut Vec<BlameEntry>, suspect: ObjectId) {
+    // Every line that has not been blamed yet on a commit, is expected to have been
+    // added when the file was added to the repository.
```

Comment:
Similar issue: the comment refers to just one of the contexts this function is called (the `Addition` case inside `match modification`).

Reply:
Thanks, this is fixed now.